src/components/products/style.jsx

```diff
@@ -54,3 +55,28 @@ export const ButtonContainer = styled.div`
   text-align: center;
   padding-bottom: 20px;
 `;
+
+export const FormWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  margin: auto;
+  margin-top: 50px;
+  background-color: rgba(64, 224, 208, 0.35);
+  color: rgb(114, 114, 236);
+  padding-top: 20px;
+  padding-bottom: 20px;
+`;
+
+export const FormGroup = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 1rem;
+  text-align: center;
+`;
+
+export const MainContent = styled.div`
```

Comment:
```suggestion
export const MainContent = styled.main`
```
A little more semantic